Makefile

```diff
@@ -122,14 +129,15 @@ container-clean:
 	docker rmi -f ${DOCKER_USER}/observability-stack:${DOCKER_TAG} \
 				  ${DOCKER_USER}/observability-stack-test-dist:${DOCKER_TAG} \
 				  ${DOCKER_USER}/observability-stack-docs-generator:${DOCKER_TAG} \
-				  ${DOCKER_USER}/observability-stack-config-service:${DOCKER_TAG}
+				  ${DOCKER_USER}/observability-stack-config-service:${DOCKER_TAG} \
```

Comment:
Why this addition? The next line isn't part of the `docker rmi` command unless I'm missing something